src/main.d.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Object whose keys are signal names and values are signal objects.
+ */
+export declare const signalsByName: { [signalName: string]: Signal }
+/**
+ * Object whose keys are signal numbers and values are signal objects.
+ */
+export declare const signalsByNumber: { [signalNumber: number]: Signal }
```

Comment:
`signalNumber` is actually a `string`: `{ '50': ... }`.

Reply:
Fixed